hearts/api/tests/test_socket_events.py

```diff
@@ -133,3 +133,96 @@ def test_create_game_when_last_player_joins(socket_clients, db):
         assert username in hands
         for u in [v for v in usernames if v != username]:
             assert u not in hands
+
+
+def test_pass_cards_add_to_pass_selections(db, socket_clients):
+    room, room_id = create_room()
+
+    usernames = ['user1', 'user2', 'user3', 'user4']
+    clients = [socket_clients.new_client() for _ in range(4)]
+    for username, client in zip(usernames, clients):
+        client.emit('join', {'room': room_id, 'username': username})  # Last user to join starts the game
+
+    '''Get the first 3 cards from user4's hand.'''
+    room = get_room(room_id)
+    game_id = str(room['game_id'])
+    game = get_game(game_id, deserialize=False)
+    current_round = game['data']['rounds'][-1]
+    hand = current_round['hands']['user4']
+    cards = hand[:3]
+
+    clients[3].emit('pass_cards', {'cards': cards})
+    game = get_game(game_id, deserialize=False)
+    assert len(game['data']['rounds'][-1]['pass_selections']) == 1
+    assert game['data']['rounds'][-1]['pass_selections']['user4'] == cards
+
+
+def test_pass_cards_add_to_pass_selections_2_users(db, socket_clients):
+    room, room_id = create_room()
+
+    usernames = ['user1', 'user2', 'user3', 'user4']
+    clients = [socket_clients.new_client() for _ in range(4)]
+    for username, client in zip(usernames, clients):
+        client.emit('join', {'room': room_id, 'username': username})  # Last user to join starts the game
+
+    room = get_room(room_id)
+    game_id = str(room['game_id'])
+    game = get_game(game_id, deserialize=False)
+    current_round = game['data']['rounds'][-1]
+
```

Comment:
Since this code setting up the unit test is used in multiple places, you should put it its own function.